coral/src/app/features/approvals/components/RequestDetailsModal.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { cleanup, render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import RequestDetailsModal from "src/app/features/approvals/components/RequestDetailsModal";
+
+const baseProps = {
+  onClose: jest.fn(),
+  onApprove: jest.fn(),
+  onReject: jest.fn(),
+};
+
+describe.only("RequestDetailsModal.test", () => {
```

Comment:
The `.only` is a leftover I would think :D

Reply:
Indeed ^^'

Reply:
[5aa2112](https://github.com/aiven/klaw/pull/611/commits/5aa2112568d76b12ef61968c0bb426a7b9436174)